tool/gravity/cli/status.go

```diff
@@ -287,10 +287,13 @@ func printStatusText(cluster clusterStatus) {
 		printAgentStatus(*cluster.Agent, w)
 	}
 
-	if len(cluster.KapacitorAlerts) != 0 {
-		fmt.Fprintf(w, "Cluster alerts:\n")
-		printClusterAlerts(cluster.KapacitorAlerts, w)
-	}
+	// Disabled due to https://github.com/gravitational/gravity/issues/2398
```

Comment:
Remove it - it is dead code. It can always be restored if necessary from history.